examples/spark/spark-driver.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I will send another pull request which will converting all the JSON spec files in spark example to YAML